applications/MeshingApplication/custom_processes/metrics_hessian_process.h

```diff
@@ -209,7 +209,7 @@ class KRATOS_API(MESHING_APPLICATION) ComputeHessianSolMetricProcess
     ///@{
 
     ModelPart& mThisModelPart;                           /// The model part to compute
-    TVarType& mVariable;                                 /// The variable to calculate the hessian
+    TVarType &mVariable;                                  /// The variable to calculate the hessian
```

Comment:
This conflicts with #3702

Reply:
1. This should not be in this PR
2. This is not how we usually do this, the current version is the correct one